browse/factory.py

```diff
@@ -30,7 +30,6 @@ def create_web_app() -> Flask:
 
     models.init_app(app)  # type: ignore
     Base(app)
-    Auth(app)
```

Comment:
This probably should not be removed. It's needed to show login status in the header (currently only on home page I think).